dummy-data/user-settings.js

```diff
@@ -84,4 +84,4 @@ const userSettings = {
   lastName: "User",
 };
 
-export default userSettings;
+export default userSettings;
```

Comment:
last line in a file should be blank line. Revert back this change

Reply:
resolved